spec/traits.dd

```diff
@@ -1930,6 +1930,30 @@ static assert(__traits(isSame, object, object));
 alias daz = foo;
 static assert(__traits(isSame, foo, daz));
 ---
+)
+        $(P isSame will also match template symbols as well as types.)
```

Comment:
I'm not 100% on this proposal, but it might be a little better.

```suggestion
        $(P isSame matches against non-instantiated templates.)
```